climate_categories/__init__.py

```diff
@@ -23,7 +23,13 @@
 cats = {}
 
 
-def _read_pickle_hier(name) -> HierarchicalCategorization:
+def _read_py_hier(name) -> HierarchicalCategorization:
+    _mod = importlib.import_module(f".data.{name}", package="climate_categories")
+    _cat = HierarchicalCategorization.from_spec(_mod.spec)
+    _cat._cats = cats
+    cats[_cat.name] = _cat
+    return _cat
+    return
     with importlib.resources.open_binary(data, f"{name}.pickle") as fd:
         _cat = from_pickle(fd)
     _cat._cats = cats
```

Comment:
This is dead code, right? Could you remove it?

Reply:
Done, thx!